test/mixin/simple-entity-mixin-test.js

```diff
@@ -0,0 +1,110 @@
+import 'd2l-fetch/d2l-fetch.js';
+import { html, fixture, expect } from '@open-wc/testing';
+
+window.D2L.Siren.WhitelistBehavior._testMode(true);
+
+async function load(el, href) {
+	let loaded;
+	const loading = new Promise(function(resolve) {
+		loaded = resolve;
+	});
+
+	async function wait(e) {
+		const promise = e.detail.promise;
+		await promise;
+		el.removeEventListener('pending-state', wait);
+		loaded();
+	}
+
+	el.addEventListener('pending-state', wait);
+	el.href = href;
+
+	await loading;
+}
+
+describe('Simple Entity Mixin Test', function() {
+
+	// this.timeout(180000);
```

Comment:
What ended up being the solution for the timeouts here?

Reply:
Oh, I was just using that to make it easier to debug the tests. Mocha will timeout the test after 10 seconds, so if you are debugging a test it can be helpful to use a longer timeout so after debugging you can let the test run to a successful/normal completion.

Reply:
Ah okay, makes sense. Mostly I was concerned that a test was taking longer than 10 seconds, which is rather long - but I do also recall that Sauce in particular can have some serious issues with doing things quickly 😢